project/Libraries.scala

```diff
@@ -5,10 +5,6 @@ object Libraries {
   def onCompile(dep: ModuleID): ModuleID = dep % "compile"
   def onTest(dep: ModuleID): ModuleID = dep % "test"
 
-  //Plugins:
-  lazy val wartRemover = "org.brianmckenna" %% "wartremover" % Versions.wartremoverV
-  lazy val androidSDKPlugin = "com.hanhuy.sbt" % "android-sdk-plugin" % Versions.androidPluginV
-
   object scala {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Object name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://www.codacy.com/app/47Degrees/scala-android/file/661377059/issues/source?bid=2339068&fileBranchId=2339109#l8)